api/client/ps.go

```diff
@@ -173,3 +224,12 @@ func (cli *DockerCli) CmdPs(args ...string) error {
 
 	return nil
 }
+
+func stringInSlice(a string, list []string) bool {
```

Comment:
There's just something similar in `pkg/stringutils` at https://github.com/docker/docker/blob/master/pkg/stringutils/stringutils.go#L44